app/__init__.py

```diff
@@ -12,11 +13,13 @@
 
 def create_app(test_config=None):
     app = Flask(__name__)
+    CORS(app)
+    app.config['CORS_HEADERS'] = 'Content-Type'
```

Comment:
Great work researching `CORS` and implementing it here -- it can be kind of a doozy the first time we have to mess with something like this! 😵